client/app/services/query.js

```diff
@@ -36,6 +40,22 @@ class Parameter {
     };
   }
 
+  get isEmpty() {
+    return isNull(this.value) || isUndefined(this.value) || (this.value === '');
+  }
+
+  getValue() {
+    if (this.isEmpty && this.useCurrentDateTime) {
+      // eslint-disable-next-line default-case
+      switch (this.type) {
+        case 'date': return moment().format('YYYY-MM-DD');
+        case 'datetime-local': return moment().format('YYYY-MM-DD HH:mm');
+        case 'datetime-with-seconds': return moment().format('YYYY-MM-DD HH:mm:ss');
+      }
+    }
+    return this.isEmpty ? '' : this.value; // normalize empty value
```

Comment:
Wouldn't it be better to use `null` as the normalized value?

Consider that we have some non string values (like numbers).

Reply:
Yes, right now I'm trying to simplify (or remove at all) `ngModel` getter/setter and move that logic to methods that can be used both with angulars `ng-model` and react components.